modules/ROOT/pages/6.4-release-notes.adoc

```diff
@@ -0,0 +1,1154 @@
+
+[source, HTML]
+----
+<p>Try to inserttextcontent here</p>
+----
+
+The new additions and improvements with the introduction of {productname} 6.4, include a new `isEditable` API been added to the `editor.selection` which now blocks the `editor.insertContent` API and `mceInsertContent` command when the contents return a `true` value if the editor detects the element is a `contentEditable = false;` element in a `contentEditable = false;` root.
+
+As a result, any `editor.selection` elements that return a `true` value when a specified element is `contentEditable = true;`, will call the new `isEditable` API to execute the new blocker.
+
+=== New `editor.dom.isEditable` and `editor.selection.isEditable` APIs that returns whether the specified node is editable
+// TINY-9462
+
+In previous versions of {productname}, elements within the editor.dom could be _modified_ regardless if they were assigned to be `contentEditable = true;` or `contentEditable = false;`.
+
+As a consequence all elements within the root `body` set to `contentEditable = false;` value, were affected by the issue.
+
+The new additions and improvements with the introduction of {productname} 6.4, include a new `isEditable` API been added to the `editor.dom` which now blocks the `editor.insertContent` API and `mceInsertContent` command when the contents return a `true` value if the editor detects the element is a `contentEditable = false;` in a `contentEditable = false;` root.
+
+As a result, any elements that return a `true` value when a specified element is `contentEditable = true;`, will call the new `isEditable` API to action the new blocker to ensure `contentEditable = false;` elements cannot be modified when they are within a root that is set to `contentEditable = false`.
+
+=== New setText and setIcon methods added to menu button and toolbar button apis
+// TINY-9268
+
+In previous versions of {productname}, it was not possible to change the `text` or `icon` of a toolbar button after it had been set for the first time. As a consequence, the menu and toolbar button API, would not extend or update the parent `icon` or `text` directly, based upon what the user had selected within the menubar or toolbar.
+
+In {productname} 6.4, two new additions were added to the menu button API.
+
+. Added `setText` and `setIcon` properties to both the toolbar button `api`, and menu button `api`.
+. Provided the menu button `api` property as an *argument* which will fetch a function that will allow menu items to change the `icon` or `text` of the parent menu button.
+
+As a result, in {productname} 6.4, menu items can change based on the parent menu button `api` properties.
+
+=== New `highlight_on_focus` option which enables highlighting the content area on focus
+// TINY-9277
```

Comment:
I don't think it's correct for this ticket. The outcome of this ticket is to add a border around the editor window so that it is visible that the editor is in focus. The border will only be rendered when the editor is in focus and with the option set to true.

Reply:
@shanmen-tiny this has been updated.